examples/reference/panes/LaTeX.ipynb

```diff
@@ -15,24 +15,23 @@
    "cell_type": "markdown",
```

Comment:
Improvements to LaTeX motivated by https://discourse.holoviz.org/t/multiline-latex-in-panel/7548/2?u=marc